addons/longer-wiwo/userscript.js

```diff
@@ -0,0 +1,23 @@
+export default async function ({ msg }) {
+  const status = $('textarea[name="status"]');
+  const left = $($("#status-chars-left").parent()[0]);
```

Comment:
Huh, asking again, why not simply `const left = $("#status-chars-left").parent();`? Not sure why you're getting the real DOM element object to then convert it to a jQuery object again, seems unnecessary?

Reply:
*facepalm* good point, didn't see that

Reply:
lol